cmake/find/FindGLFW.cmake

```diff
@@ -0,0 +1,83 @@
+# ----------------------------------------------------------------------
+# FindGLFW.cmake - GLFW CMake module
+#
+# Copyright (C) 2016 Alexandre Pretyman. All rights reserved.
+#
+# Defines GLFW target if found
+#
+# ----------------------------------------------------------------------
+
+
+
+if(TARGET "GLFW")
+  return()
+endif()
+
+hunter_status_debug("GLFW_ROOT: ${GLFW_ROOT}")
+
+find_path(
+  GLFW_INCLUDE_DIR
+    NAMES
+      GLFW/glfw3.h
+    PATHS
+    "${GLFW_ROOT}/include"
+    NO_DEFAULT_PATH
+)
+set(GLFW_FOUND FALSE)
+find_library(
+    GLFW_LIBRARY
+      glfw3
+    PATHS
+      "${GLFW_ROOT}/lib"
+    NO_DEFAULT_PATH
+)
+if(GLFW_LIBRARY)
+  unset(_glfw_dependencies)
+  if(APPLE)
+    list(APPEND _glfw_dependencies "-framework Cocoa")
+    list(APPEND _glfw_dependencies "-framework CoreVideo")
+    list(APPEND _glfw_dependencies "-framework IOKit")
+    list(APPEND _glfw_dependencies "-framework OpenGL")
+  elseif(UNIX) # linux 
+    find_package(OpenGL REQUIRED)
+    list(APPEND _glfw_dependencies "${OPENGL_gl_LIBRARY}")
+
+    #Below taken from https://github.com/PixarAnimationStudios/OpenSubdiv/blob/master/cmake/FindGLFW.cmake
+    find_package(X11 REQUIRED)
```

Comment:
Sorry Ruslan,

I have not updated the pull request yet, this Find script won't be used. The The glfw3Config.cmake generated will be used.